internal/charmstore/migrations.go

```diff
@@ -138,6 +142,36 @@ func addSupportedSeries(db StoreDatabase) error {
 	return nil
 }
 
+// handleDevelopmentEntities adds the Development field to all entities and
+// sets up ACLs for development revisions.
+func handleDevelopmentEntities(db StoreDatabase) error {
+	logger.Infof("adding development field to all entities")
+	if _, err := db.Entities().UpdateAll(nil, bson.D{{
```

Comment:
This should probably only update those that don't have a development field already. That way it's idempotent and its fairly easy to do.